.github/workflows/oas.yml

```
@@ -0,0 +1,34 @@
+name: OAS
+
+on:
+  push:
+    branches:
+      - main
```

The main branch in open-product i think is `master` right?
```suggestion
      - master
```